deserializer.go

```diff
@@ -4,14 +4,17 @@ import (
 	"bytes"
 	"encoding/gob"
 	"regexp"
-	"strconv"
 	"strings"
 	"time"
 
-	"github.com/armon/go-radix"
+	radix "github.com/armon/go-radix"
```

Comment:
It compiled because the package name is actually `radix` - https://github.com/armon/go-radix/blob/master/radix.go#L1

I'm confused as to why it wouldn't compile without the alias ...